pkg/plugins/klogs/instance/parser/parser.go

```diff
@@ -128,7 +128,7 @@ func (s *SQLParser) parsePredicate(p *Predicate) string {
 	}
 
 	if p.Not != nil {
-		return fmt.Sprintf("NOT ( %s )", s.parsePredicate(p.Not))
+		return fmt.Sprintf("NOT %s", s.parsePredicate(p.Not))
```

Comment:
I removed the brackets here (and in the related tests), because we already add brackets for subexpressions. These brackets aren't necessary, because of SQL operator precedence, try yourself with:
```bash
kubectl exec -it chi-clickhouse-replicated-0-0-0 -- clickhouse-client -h 127.0.01 -q "SELECT true = false, NOT true = false, true = true" 
0       1       1
```